CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 # Changelog
 
+## v0.31.1
+
+*March 27th, 2019*
+
```

Comment:
Don't you want to write short description for the release?

Reply:
Is this OK:
This release contains improvements for the mempool that reduce the bandwidth requirements for nodes 
(some numbers are provided below).   
We also include a fix to a memory leak in the mempool and RPC HTTP servers can now be run over TLS.
Besides that it contains minor improvements, bugfixes, and code cleanups.

Reply:
```
This release contains a major improvement for the mempool that reduces the amount of data sent by ~30% (see the numbers below). It also fixes a memory leak in the mempool and gives you ability to run HTTPs (over TLS) RPC server by providing certificate and key in the config.
```
I'd remove `Besides that it contains minor improvements, bugfixes, and code cleanups.`